src/services/codefixes/inferFromUsage.ts

```diff
@@ -668,6 +668,10 @@ namespace ts.codefix {
             }
         }
 
+        function inferTypeFromExpressionStatement(node: Expression, usage: Usage): void {
+            addCandidateType(usage, isPropertyAccessExpression(node) ? checker.getAnyType() : checker.getVoidType());

```

Comment:
I think it would be safer to only infer `void` from call expressions and give `any` for everything else.